src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -561,6 +562,18 @@ class ArnoldRenderer : public IECoreScenePreview::Renderer
 				}
 				return;
 			}
+			else if( name == g_logFileNameOptionName )
+			{
+				if( value == NULL )
+				{
+					AiMsgSetLogFileName( "" );
```

Comment:
I definitely get the feeling that the full API options are a bit much for users (which is probably why kick just has 0-6 instead of each individually). In my personal render debugging so far, I seem to have settled on v2 as my default, and v6 for re-renders when anything goes wrong. That's partly because I have no intuition for what I lose/gain for the intermediate numbers, but also because much of my debugging thus far has been about ill defined plugin paths and such, so I just assumed that would require top verbosity to fix...

As for controlling logs vs console separately, I guess that could make sense. When I go up to v6 I get a bunch of stuff that seems to make more sense in the log file living alongside the renders than it does in the Farm stream (where it clutters and slows down the Qube user experience). Maybe a different Farm manager wouldn't suffer from that though? Or maybe its useful for the details to go to both for kibana/graphana reporting purposes?

@lento @don-at-cinesite @danieldresser @mattigruener any opinions on this stuff?